CONTRIBUTING.md

```diff
@@ -47,6 +47,17 @@ The following topics should generally not be proposed for discussion as they are
 * Platform-Specifics which can be accomplished without changing Xamarin.Essentials
 * Avoid clutter posts like "+1" which do not serve to further the conversation
 
+#### Guiding Principles for New Features
+
+Any proposals for new feature work and new APIs should follow the spirit of these principles:
+
+ * APIs should be simple, direct, and generally implemented with static classes and methods whenever practical
+ * Usage of interfaces is to be strictly avoided - APIs should be simple and performant
+ * Custom UI should be entirely avoided
+ * UI code is only allowable in cases where the platform provides an implementation (eg: Browser, Email Composer, Phone Dialer, etc)
+ * New features should have native API's available to allow implementation on a reasonable subset of the supported platforms, especially  (iOS, Android, UWP)
+ * No new external dependencies should generally be added to support implementation of new feature work (there can be exceptions but they must be thoroughly considered for the value being added)
```

Comment:
```suggestion
 * No new external dependencies should be added to support implementation of new feature work (there can be exceptions but they must be thoroughly considered for the value being added)
```

No need for the "generally" as your comment in parentheses indicates that this rule may be broken in special cases.